src/status.py

```diff
@@ -7,6 +7,14 @@
 
 import ops
 
+PRIORITY_MAP = {
```

Comment:
Would it make sense to open a ticket to request this in ops?

Reply:
That sounds like a good idea :D I have seen that there is a [similar feature](https://ops.readthedocs.io/en/latest/#ops.CollectStatusEvent) but not exactly.